src/cpp/r/R/Options.R

```diff
@@ -143,3 +143,5 @@ options(profvis.keep_output = TRUE)
 # indicate that we're not in a notebook by default
 options(rstudio.notebook.executing = FALSE)
 
+# find in files: exclude things in platform-specific subdirs
+options(rstudio.find_in_files.excludes = R.version$platform)
```

Comment:
Does setting this in `options.R` (which runs after all of R's startup code) make it impossible for users to set it as a sticky preference?

Reply:
Good point -- perhaps we should test if it's `NULL` first.